test/nn/test_lazy_modules.py

```diff
@@ -583,9 +583,9 @@ def forward(self, x):
 
     @suppress_warnings
     def test_optimizer_pass(self):
+        # Add Adamax and RAdam when #118230 and #117836 are complete
```

Comment:
This list doesn't seem to include all the optimizers btw @mikaylagawarecki

Reply:
oh thanks, is there a global list of optimizers I can replace this with?

Reply:
There is optim_db in common_optimizers :)